docs/integrations/microsoft-azure/microsoft-dynamics365-customer-insights.md

```diff
@@ -0,0 +1,59 @@
+
+* Logs collection from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) using our [Azure Event Hubs source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/).
+
+When you configure the event hubs source or HTTP source, plan your source category to ease the querying process. A hierarchical approach allows you to make use of wildcards. For example: `Azure/Dynamic365/Logs`
+
+### Prerequisites
+
+Follow the [prerequisites](https://learn.microsoft.com/en-us/dynamics365/customer-insights/diagnostics#prerequisites) before configuring the collection setup. We will be using Event hub as a destination resource, and more details about its limitations and permissions are described [here](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings?tabs=portal#destination-limitations).
+
+### Configure logs collection
+
+In this section, you will configure a pipeline for shipping diagnostic logs from Azure Monitor to an Event Hub.
+
+1. Create an Event Hubs namespace as described in [step 2 of the prerequisites section](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/#prerequisites). Here, you don't have to create an Event Hub Instance in step 3 since `Microsoft Dynamics 365 Customer Insights` automatically creates the below two Event Hubs:
+
+    * **insight-logs-audit**. It contains audit events.
+    * **insight-logs-operational**. It contains operational events.
+
+2. Create a [Shared Access Policy](https://docs.microsoft.com/en-us/azure/governance/policy/overview) for the entire namespace with the `Listen` claim or you can use the existing default `RootManageSharedAccessKey` policy. Copy the `Primary key` associated with this policy. For more details, refer to the [steps 4 and 5 of the prerequisites section](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/#prerequisites).
+
+3. Create two Azure Event Hubs Sources using the instructions described [here](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/#create-an-azure-event-hubssource). You can add both of them to the same Hosted Collector. Provide `insight-logs-audit` and `insight-logs-operational` as `Event Hubs Instance Name` in the two Azure Event Hubs Sources, respectively.
+
+4. To create the Diagnostic settings in the Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-us/dynamics365/customer-insights/diagnostics#set-up-diagnostics-with-azure-monitor).
+   1. Choose Event Hub as the`Resource type`.
+   1. Select the Event Hub's `Subscription` name, `Resource group` name, and `Resource` name for the destination resource.
```

Comment:
```suggestion
1. Create an Event Hubs namespace as described in [step 2 of the prerequisites section](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/#prerequisites). Here, you don't have to create an Event Hub Instance in step 3 since `Microsoft Dynamics 365 Customer Insights` automatically creates the below two Event Hubs:
    * **insight-logs-audit**. It contains audit events.
    * **insight-logs-operational**. It contains operational events.
2. Create a [Shared Access Policy](https://docs.microsoft.com/en-us/azure/governance/policy/overview) for the entire namespace with the `Listen` claim or you can use the existing default `RootManageSharedAccessKey` policy. Copy the `Primary key` associated with this policy. For more details, refer to the [steps 4 and 5 of the prerequisites section](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/#prerequisites).
3. Create two Azure Event Hubs Sources using the instructions described [here](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/#create-an-azure-event-hubssource). You can add both of them to the same Hosted Collector. Provide `insight-logs-audit` and `insight-logs-operational` as `Event Hubs Instance Name` in the two Azure Event Hubs Sources, respectively.
4. To create the Diagnostic settings in the Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-us/dynamics365/customer-insights/diagnostics#set-up-diagnostics-with-azure-monitor).
   1. Choose Event Hub as the `Resource type`.
   1. Select the Event Hub's `Subscription` name, `Resource group` name, and `Resource` name for the destination resource.
```